### DISCUSSION TITLE
pip install -U eel: auto-py-to-exe 2.31.0 requires Eel==0.14.0, but you have eel 0.16.0 which is incompatible.

### DISCUSSION THREAD

**Discussion by ackhh** [brentvollebregt/auto-py-to-exe#370] [General]
Is that an issue? auto-py-to-exe seems to be working fine with eel 0.16.0

**Comment by brentvollebregt** [comment]
I used a set version as a change was made to Eel a while ago that broke this project. If 0.16.0 is working ok, it can be bumped up.

**Reply by brentvollebregt** [reply]
I've just put a change on master that allows 0.11.0 and up - this will go out with the next release
